.evergreen/config.yml

```diff
@@ -161,6 +189,7 @@ buildvariants:
   tasks:
   - build-and-test-and-upload
   - build-and-test-shared-bson
+  - build-and-test-java
```

Comment:
I'll remove this - it doesn't pass

Reply:
Which part doesn't pass.  Just Windows?  If so, can we just disable Windows?  IIUC this will disable all Java binding testing.

Reply:
Thats just the windows build task list. There's already a ticket to fix / test windows and the java driver (CDRIVER-3131) that will re add it.